module3/lessons/week3/CreatingRelatedResources.md

```diff
@@ -0,0 +1,349 @@
+---
+layout: page
+title: Creating and Reading Related Resources
+---
+
+<section class='instructor-notes' markdown='1'>
+
+The completed code for this lesson is on the create-related-resource branch of the [MvcMovieStarter](https://github.com/turingschool-examples/MvcMovieStarter/tree/create-related-resource)
+
+</section>
+
+## Learning Goals
+- Use a form to create a one-to-many relationship
+- Create Index Page for a Resource with related data
+
+### Demo
+
+Your instructor will demonstrate what our MvcMovie application will look like after today's lesson.  Our goal today is to:
+* Add Movie Reviews to our database
+* Include a Movie's reviews on it's show page
+* Use a form to create new reviews for a movie
+
+## Nested Resources
+
+So far, we have been working with a single resource - Movie.  This makes our routes relatively straight forward. We learned about RESTful routes and implemented them into our project.
+
+<section class='call-to-action'>
+<strong>To Review</strong>: List all the RESTful routes for Movie in your notebook.
+</section>
+
+RESTful routes can also be used for more complex data relationships.  
+
+We are going to be updating our application so that we have a database similar to the diagram below:
+![](/assets/images/module3/Week3/MovieReviewERD.png)
```

Comment:
Love that you have the diagram here! Not sure if it's worth adding now, but for future ERD diagrams I think would be nice to show the lines that say 1 to 1, one-to-many, etc.